views/forms/index.ejs

```diff
@@ -1,18 +1,20 @@
 <%- contentFor('title') %>Form
 <%- contentFor('body') %>
-<h1>Forms Page</h1>
+<h1>AB-540 Application</h1>
 
-
-<a href="/forms/new">Add a New Forms</a>
-<p>Forms for Undocument Immigrants</p>
-<ul>
+<b>What is a AB-540?</b><br>
+<a>AB-540 allows undocmented people to pay instate tuition in California instead of paying a out of state fees.</a>
+ <p align="right"><a href="https://ab540.com/What_Is_AB540_.html">show more!>></a></p>
+<b>AB-540 Applications</b><br>
+<b>Find your school below:</b>
+<ul align="center">
   <% for (let record of records) { %>
     <li>
-      <a href="/forms/<%= record.id %>">
-        <% if (record.logo) { %>
-          <img src="<%= assetUrl(record.logo) %>" />
-        <% } %>
-        <%= record.name %> <%= record.year %><br><%= record.school %><br><%= record.url %><br><%= record.lang %></a><br>
-      </li>
+      <a href="/forms/<%= record.id %>"><%= record.name %> <%= record.year %></a><p></p>
+      <% if (record.logo) { %>
+        <img src="<%= assetUrl(record.logo) %>" /><p></p>
```

Comment:
On lines 13 and 15, you use an empty <p></p> paragraph to add space between the image and the surrounding text and buttons.

Can you instead add a class to the tags, and add margin and padding styles to the class, instead?